package.json

```
@@ -1,13 +1,16 @@
 {
-	"name": "o-errors",
-	"private": true,
+	"description": "A decoupled events-based mechanism for components to report errors and an API for products to report client-side errors.",
+	"keywords": ["error", "errors"],
+	"name": "@financial-times/o-errors",
```

I think we have a bug in origami-build-tools because the tests for this pull-request are passing but the package.json does not define an entrypoint for the javascript.

I've suggested a `browser` field and not `main` field because this component's code does not support running in NodeJS
```suggestion
	"name": "@financial-times/o-errors",
	"browser": "main.js",
```

---

!!!!